rpc_jobs/multi_node_aio.yml

```diff
@@ -34,6 +34,24 @@
       - 'OnMetal-Multi-Node-AIO_{series}-{context}-{trigger}'
       - 'OnMetal-Multi-Node-AIO-Merge-Trigger_{series}'
 
+- project:
+    name: 'Leapfrog-Multi-Node-AIO-Jobs'
+    series:
+      - kilo:
```

Comment:
Agreed with @git-harry . Series should be newton14.1, even if it's a new project.